facades/sort_calories_facade.js

```diff
@@ -1,9 +1,8 @@
 var Recipe = require('../models').Recipe;
 var RecipeHelper = require('../helpers/recipe_helper');
 var RecipeSerializer = require('../serializers/recipe_serializer');
-var pry = require('pryjs')
 
-module.exports = class SortFacade {
+module.exports = class SortCaloriesFacade {
```

Comment:
only because im now using this facade for both sorting calories and times we might want to name this sort recipes facade